.docker/public-nginx.conf

```diff
@@ -63,7 +66,10 @@ server {
     listen 0.0.0.0:8082;
 
     location = /favicon.ico { access_log off; log_not_found off; }
-    location = /robots.txt { access_log off; log_not_found off; }
```

Comment:
Might want to retain `access_log` directive?

Reply:
I'd wait for it to produce too much noise? The log would be the only trace that this is being hit.